ui/src/components/connections/ConnectionDeleteModal.tsx

```diff
@@ -0,0 +1,78 @@
+import { Checkbox, Divider, Modal, Space, Typography, message } from "antd";
+import { CheckboxChangeEvent } from "antd/es/checkbox";
+import { useState } from "react";
+import { deleteConnection } from "src/adapters/api/connections";
+
+import { ReactComponent as IconClose } from "src/assets/icons/x.svg";
+import { useEnvContext } from "src/contexts/env";
+
+export function ConnectionDeleteModal({
+  callback,
```

Comment:
This should probably be `onDelete` so it's clear how this prop would be defined.